src/implementations/twig/components/social-media-follow/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ecl/twig-component-social-media-follow",
+  "private": true,
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "3.0.0",
+  "description": "ECL Social Media Follow",
+  "dependencies": {
+    "@ecl/twig-component-icon": "^3.0.0",
```

Comment:
Why also icon in the dependencies? We import link, then link has icon as a dependency.